scdrs/data/res/toydata_gs_human.scdrs_ct.cell_type

```diff
@@ -1,3 +1,3 @@
 	n_cell	n_ctrl	assoc_mcp	assoc_mcz	hetero_mcp	hetero_mcz
-causal_cell	10.0	20.0	0.04761905	12.297529	1.0	1.0
-non_causal_cell	20.0	20.0	0.9047619	-1.1364214	1.0	1.0
+causal_cell	10.0	20.0	0.04761905	11.596751	0.14285715	1.3146176
```

Comment:
the change is due to the difference of dealing covariates between versions, is this correct?

Reply:
Yes.